Web/Html Assignment 1/Assignment-1.html

```diff
@@ -0,0 +1,177 @@
+						<option>6</option> 
+					</select> months
+				</td>
+			</tr>
+			<tr>
+				<td align="right">Current Industry</td>
+				<td><select name="Select"> 
+						<option>select</option>
+						<option> metacube</option> 
+						<option>Xyz</option> 
+					</select>
+				</td>	
+			</tr>
+			<tr>
+				<td align="right">Functional Area</td>
+				<td><select name="Select"> 
+						<option>select</option>
+						<option> java</option> 
+						<option>Salesforce</option> 
+					</select>
+				</td>	
+			</tr>
+			<tr>
+				<td> </td>
+				<td> <a href = "#" > What is my Functional Area and Industry? </a></td>
+			</tr>
+			<tr>
+				<td align="right">Key Skills</td>
+				<td><textarea rows="3" cols="50" name="keyskills" placeholder="Please enter your areas of expertise / specialization"></textarea></td>
+			</tr>
+			
+			<tr>
+			<td align="right"> Resume HeadLine</td>
+			<td><textarea rows="1" cols="100" placeholder="Describe your professional background in one line"></textarea> </td>
+			</tr>
+			<tr>
+				<td colspan = "2"><hr></td>
+			</tr>
+			<tr>
+			<tr colspan=2>
+				<td><b>Education Details</b></td>
+			</tr>
+				<td align="right">Basic Graduation</td>
+				<td><select name="BasicGraduation"> 
+						<option>select</option>
+						<option> metacube</option> 
+						<option>Xyz</option> 
+					</select>
+				</td>
+			<tr>
+			</tr>
+				<td align="right">Post Graduation</td>
+				<td><select name="PostGraduation"> 
+						<option>select</option>
+						<option> metacube</option> 
+						<option>Xyz</option> 
+					</select>
+				</td>
+			<tr>
+			</tr>
+				<td align="right">Doctrate</td>
+				<td><select name="Doctrate"> 
+						<option>select</option>
+						<option> metacube</option> 
+						<option>Xyz</option> 
+					</select>
+				</td>
+			<tr>
+			</tr>
+			<tr>
+				<td colspan = "2"><hr></td>
+			</tr>
+			<tr>
+			<tr colspan=2>
+				<td><b>Upload Resume</b></td>
+			</tr>
+			<tr>
+				<td align="right"> Attach Resume </td>
+				<td><input type="file" name="file"> </td>
+			</tr>
+			<tr colspan=2>
+				<td><b>Mailer and  Privacy Settings</b></td>
+			</tr>
+			<tr>
+			<tr>
+				<td><input type="checkbox" name="jobAlertCheck"> Job Alerts</td>
+				<td><input type="checkbox" name="importantNotificationCheck"> Important Notifications</td>
+			</tr>
+			<tr>
+				<td><input type="checkbox" name="jobAlertCheck"> Naukri Fastforward E-Mails</td>
+				<td><input type="checkbox" name="importantNotificationCheck"> Communication from client (Voice / Calls / SMS)</td>
+			</tr>
+			<tr>
+				<td><input type="checkbox" name="jobAlertCheck"> Naukri FastForward Calls/SMS</td>
+				<td><input type="checkbox" name="importantNotificationCheck"> Other promotions / Special Offers</td>
+			</tr>
+			<tr>
+				<td colspan = "2"><hr></td>
+			</tr>
+			<tr>
+			<td colspan = "2"><input type="checkbox" name="accept">I have read & understood, and agree to the <a href="#">Terms and Conditions</a> governing the use</td>
```

Comment:
You should use :- "&amp;"    instead of directly using "&" character as it could be treated as a special character accidently and the rest of the text might not get rendered properly.